libs/core/message-strip/message-strip.component.ts

```diff
@@ -164,22 +158,28 @@ export class MessageStripComponent implements OnInit, OnChanges, CssClassBuilder
         this.setScreenReaderTexts();
     }
 
+    /** @hidden */
+    ngAfterContentInit(): void {
```

Comment:
@InnaAtanasova do you think this is safe to only check once on init rather than `ngAfterContentChecked`?